services/app-api/forms/mcpar.json

```
@@ -1338,35 +1338,90 @@
                     "validation": "radio",
                     "props": {
                       "label": "C2.V.4 Provider type",
-                      "hint": "What is the provider type? Select one or add free text if you select “Other”.",
+                      "hint": "Enter the core provider type the standard applies to. If the standard applies to multiple provider types, create a measure for each those provider types. If you wish to specify the provider type, select the core type most suitable and then add the more specific provider type under “Specialist details”.",
```

typo? either add "of" or remove "those" and make "types" singular
```suggestion
                      "hint": "Enter the core provider type the standard applies to. If the standard applies to multiple provider types, create a measure for each of those provider types. If you wish to specify the provider type, select the core type most suitable and then add the more specific provider type under “Specialist details”.",
```

---

Great catch! Mentioned this to Kim and we're going with "of" for now :D.